### ISSUE TITLE
Weather info on home screen and in agenda NOT displaying/working

### ISSUE THREAD

**Issue by saVagebecKa** [8bitPit/Niagara-Issues#3039]
### Describe the bug

I have Niagara Launcher Pro. I am on a brand new Motorola Edge 5g 2024, Android 14. 
Ever since I installed Niagara Launcher Pro on my phone, I have not been able to have ANY weather info working out displaying on my home screen AND in the agenda. 

I have gone thru the website, the FAQs, and tried all possible solutions to figure out this issue, but have not been successful. (And as you can see from the click widget I placed above, it has working weather info just fine). 
I've tried using current location. I've tried using fixed location. I have tried fixed location with just my city/state, AND with my full address, AND with my actual coordinates from Google maps entered. I've downloaded the open weather app. I have all permissions for Niagara Launcher accepted. I have my location on.

I have NO issues with my phone requesting or receiving any location data, for any other apps. 

Refer to screenshots for additional info:

![Screenshot_20241130-022050_Niagara Launcher](https://github.com/user-attachments/assets/f466922a-378c-4f80-a439-684ecc7ef6c3)
![Screenshot_20241130-021651_Niagara Launcher](https://github.com/user-attachments/assets/9fc0bb41-1071-4089-a19a-dff44add0799)
![Screenshot_20241130-021840_Niagara Launcher](https://github.com/user-attachments/assets/e72d3e8a-a285-41cd-9fdd-e884ee35a881)
![Screenshot_20241130-022118_Niagara Launcher](https://github.com/user-attachments/assets/3e70f50d-5614-47f5-8839-e4d4826a8474)
![Screenshot_20241130-022723_Niagara Launcher~2](https://github.com/user-attachments/assets/f59c4afc-45c6-42dd-8cf2-ed700189d796)
![Screenshot_20241130-022632_Niagara Launcher~2](https://github.com/user-attachments/assets/eb1da411-e597-44bb-acec-d9deb289451d)
![Screenshot_20241130-022606_Niagara Launcher~2](https://github.com/user-attachments/assets/5c6044a0-cb24-46b5-8464-b2fde9789a15)
![Screenshot_20241130-022549_Niagara Launcher~2](https://github.com/user-attachments/assets/a637ab22-6824-4c73-84b4-aa7f70c58f1d)
![Screenshot_20241130-022434_Niagara Launcher~2](https://github.com/user-attachments/assets/cb16b6d6-522c-48c4-9965-ef0638980d94)
![Screenshot_20241130-024236_Niagara Launcher](https://github.com/user-attachments/assets/6a874021-39fe-43cb-a10e-9cca43320ed3)

### Steps to reproduce

1. Go to at-a-glance Niagara widget and tap for agenda. 
2.  See "can't show weather data - tap for more info" message at top of agenda and tap.
3. Error message:  
Can't show weather

We have been notified about this error.

Error message: NoWeather DataException(st missing)

Error occurred with Niagara Launcher v1.12.17 on a Motorola device which runs Android 14 on 2024-11-30T09:57:27Z. My crash user id is '1da230aa-4686-4f72-a49c-3e9db69442a2'.
(Error message copied and pasted exactly as-is)

### Does the issue also occur with other 3rd-party launchers?

Not applicable

### Device name

Motorola Edge 5g 2024 (bought and activated Nov 15 2024)

### Android version

Android 14

### Screenshots / Screen recordings

![Screenshot_20241130-022050_Niagara Launcher](https://github.com/user-attachments/assets/f466922a-378c-4f80-a439-684ecc7ef6c3)
![Screenshot_20241130-021651_Niagara Launcher](https://github.com/user-attachments/assets/9fc0bb41-1071-4089-a19a-dff44add0799)
![Screenshot_20241130-021840_Niagara Launcher](https://github.com/user-attachments/assets/e72d3e8a-a285-41cd-9fdd-e884ee35a881)
![Screenshot_20241130-022118_Niagara Launcher](https://github.com/user-attachments/assets/3e70f50d-5614-47f5-8839-e4d4826a8474)
![Screenshot_20241130-022723_Niagara Launcher~2](https://github.com/user-attachments/assets/f59c4afc-45c6-42dd-8cf2-ed700189d796)
![Screenshot_20241130-022632_Niagara Launcher~2](https://github.com/user-attachments/assets/eb1da411-e597-44bb-acec-d9deb289451d)
![Screenshot_20241130-022606_Niagara Launcher~2](https://github.com/user-attachments/assets/5c6044a0-cb24-46b5-8464-b2fde9789a15)
![Screenshot_20241130-022549_Niagara Launcher~2](https://github.com/user-attachments/assets/a637ab22-6824-4c73-84b4-aa7f70c58f1d)
![Screenshot_20241130-022434_Niagara Launcher~2](https://github.com/user-attachments/assets/cb16b6d6-522c-48c4-9965-ef0638980d94)
![Screenshot_20241130-024236_Niagara Launcher](https://github.com/user-attachments/assets/6a874021-39fe-43cb-a10e-9cca43320ed3)

**Comment by lixiansen007-77** [comment]
Has it been resolved?

**Comment by ghost** [comment]
> Has it been resolved?

No, I am having this issue on the latest version of Niagara launcher
